docs-ref-services/security-keyvault-keys-readme-pre.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Azure Key Vault Key client library for Java
-keywords: Azure, java, SDK, API, azure-security-keyvault-keys, keyvault
+keywords: Azure, , SDK, API, azure-security-keyvault-keys, keyvault
```

Comment:
Missing `java`. This is a script bug. We have fixed the bug in the script, but we have to fix them manually this time.

Reply:
Same for the below readmes.